nemo/core/classes/modelPT.py

```diff
@@ -1400,3 +1403,76 @@ def update_save_restore_connector(cls, save_restore_connector):
             cls._save_restore_connector = save_restore_connector
         else:
             setattr(cls, '_save_restore_connector', save_restore_connector)
+
+    def _setup_nsys_profiling(self):
```

Comment:
Private methods don't show up in core docs, we should maybe force sphinx to show this function to users

Reply:
Yes, we can add docs for this.